be/test/exprs/string_functions_test.cpp

```diff
@@ -0,0 +1,111 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
you should add run this test to run-ut.sh file to make regression test can run this test

Reply:
OK